tests/helpers.py

```diff
@@ -55,7 +55,6 @@ def search(self):
         """
         Override for abstract base method.
         """
-        pass
```

Comment:
Same question as above for keeping this function definition if it's now empty.

Reply:
Still needed...the search function in the base class is defined using `@abstractmethod`, so we need to override it, even if its with an empty function.